src/adaptix/_internal/model_tools/introspection/dataclass.py

```diff
@@ -51,10 +51,10 @@ def _create_inp_field_from_dc_fields(dc_field: DCField, type_hints):
 
 
 if HAS_PY_310:
-    def _get_param_kind(dc_field: DCField) -> ParamKind:
+    def _get_param_kind(dc_field: DCField, /) -> ParamKind:
         return ParamKind.KW_ONLY if dc_field.kw_only else ParamKind.POS_OR_KW
 else:
-    def _get_param_kind(dc_field: DCField) -> ParamKind:
+    def _get_param_kind(_dc_field: DCField, /) -> ParamKind:
```

Comment:
well, i don't think that IDE warnings are a good reason for making argument pos-only and such renaming